classes/Controller/ThanksController.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace Claromentis\ThankYou\Controller;
+
+use Claromentis\Core\Config\Exception\DialogException;
+use Claromentis\Core\Http\RequestData;
+use Claromentis\Core\Http\RequestDataTokenException;
+use Claromentis\Core\Http\TemplaterCallResponse;
+use Claromentis\Core\Localization\Lmsg;
+use Claromentis\Core\Security\Exception\AccessDeniedException;
+use Claromentis\Core\Security\SecurityContext;
+use Claromentis\Core\Widget\Sugre\SugreUtility;
+use Claromentis\ThankYou\Api;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Log\LoggerInterface;
+
+class ThanksController
```

Comment:
We usually separate admin panel controllers from front end controllers, and name or namespace them accordingly.

If this class is exclusively for admin panel pages, it can be called `ThankYouAdminController`. If it's a mix between front end and admin, it would be clearer to other developers to have these split up (perhaps a `Claromentis\ThankYou\Controllers\Admin` namespace).

Reply:
Fixed in 0af5553d12a3ee031b3bbc7f1cdb02fa0932cfaa
Worth a review.

Reply:
All of the controller classes should have the `Controller` suffix, I'm not sure why these were removed.

Reply:
Considering the namespace/folder structure is `Controllers\blahblah`, I figured the "Controllers" part was redundant as it was fairly clearly inferred. If I started to see clashes with imported classnames elsewhere I would have probably used `Controllers\Admin::class` rather than just `Admin::class` to help clarify.

Made the change anyway :), fixed in 0fca0d80331a7ad6af4b7e16e00d4f3818d3badd

Reply:
It also helps inform the obfuscator that the class is indeed a controller (though because this module is open source, it's not obfuscated). Still, good better for consistency and searchability.

Cheers!